src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs

```diff
@@ -101,11 +106,13 @@ internal static partial class NetSecurityNative
         [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_AcceptSecContext")]
         internal static extern Status AcceptSecContext(
             out Status minorStatus,
+            SafeGssCredHandle acceptorCredHandle,
             ref SafeGssContextHandle acceptContextHandle,
             byte[] inputBytes,
             int inputLength,
             ref GssBuffer token,
-            out uint retFlags);
+            out uint retFlags,
+            out bool isNtlmUsed);
```

Comment:
What is the size of this bool? Consider adding a `[MarshalAs]` to make this clear.

```suggestion
            [MarshalAs(UnmanagedType.Bool)] out bool isNtlmUsed);
```

https://github.com/dotnet/corefx/blob/01e67503404ccf0a8dc8deb69f285d8ea0f42ce6/src/Native/Unix/System.Net.Security.Native/pal_gssapi.c#L308

Reply:
most pal use 'int' AFAIK unless needed otherwise.

Reply:
The other places in our PAL use 'int32_t' as well. And those declarations don't use any "MarshalAs' syntax either to marshal the 32-bit integer into a managed bool. E.g.

https://github.com/dotnet/corefx/blob/2d713265d5aecd5fe34099042e13fe7ba54dcdd1/src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs#L99

I'm going to use the same pattern for now and not have any "MarshalAs" annotations. But I'll ask around to see if there is any value in adding those to here and other places in the PAL.

Reply:
> The other places in our PAL use 'int32_t' as well. And those declarations don't use any "MarshalAs' syntax either to marshal the 32-bit integer into a managed bool.

I'd rather have consistency, so I'm on board for that.

>  I'll ask around to see if there is any value in adding those to here and other places in the PAL.

The value is only readability.`bool` marshals by default as `UnmanagedType.Bool`, which is for the win32 32-bit `BOOL` type. It's an easy thing to miss if you're meaning to p/invoke a 1-byte bool. Adding an explicit `MarshalAs` lets the reader know you've considered the behavior. Maybe @JeremyKuhne has an opinion here.

Reply:
We, as a general rule, don't put the attributes on the imports that match the defaults. It is preferable to not use the c# `bool` and be explicit about the underlying type to streamline the generated IL, but isn't critical. If you're using `BOOL` we have an `Interop.BOOL` enum (and `BOOLEAN`) that can be used for clarity/efficiency.